opentelemetry/src/main/java/io/grpc/opentelemetry/GrpcOpenTelemetry.java

```diff
@@ -152,6 +154,7 @@ public void configureChannelBuilder(ManagedChannelBuilder<?> builder) {
     InternalManagedChannelBuilder.addMetricSink(builder, sink);
     InternalManagedChannelBuilder.interceptWithTarget(
         builder, openTelemetryMetricsModule::getClientInterceptor);
+    builder.intercept(openTelemetryTracingModule.getClientInterceptor());
```

Comment:
We should avoid enabling tracing until we are happy with it. Are we happy with it right after this PR?

You could add a package-private builder option to allow enabling it for testing and such.

Reply:
No I don't think we should make the API public and stable right away.

Let me split the PR and have interop test in a separate PR.
The interop test needs it to be public API though, how about we have environmental variable and disable by default ?

Reply:
Env var is fine, but you can't change env vars from Java. You could use system properties instead.

But I'd probably just make the package-private method on the builder and then make an InternalGrpcOpenTelemetry class that exposes the method for the test to use.